en/salesforce/src/docs/TEMPLATE_ms_composer_custom_expression_editor.adoc

```diff
@@ -1,85 +1,304 @@
-= Build Formulas for a Flow
+= Building Formulas for Flows
 
-You can manipulate data in Composer by building expressions and applying functions. With the *Custom Expression* editor, you can use DataWeave functions for number, string, or date manipulation using simple math, truncating, capitalizing or setting a date.
+You can manipulate data in Composer by building _expressions_ and applying _functions_. With the *Custom Expression Editor*, you can create expressions that include functions and mathematical operators. For example, you can truncate words, capitalize phrases, and set the date.
 
 == Before You Begin
 
-* Only functions that accept data types as inputs appear.
-* Only data pills of the same data type as the field appear. This does not apply to string fields where automatic type casting is already supported.
-* Functions can be nested. The input parameters of a function can be another function call.
-* The data pills and formulas displayed in the if/else block for the value are dependent on the data type of the selected field.
+//More content may be added - to be determined later
+* You can nest functions. You can use the input parameters of another function call for your function.
+* Composer functions used in Composer are built using https://docs.mulesoft.com/dataweave/2.3/dw-operators[DataWeave].
```

Comment:
This is just being picky, but when I see "Before You Begin" I expect a list of stuff I have to do rather than stuff I have to know. Would it be too odd to use wordings like these?
```suggestion
* Keep in mind that you can nest functions [[and use? by using?]] the input parameters of another function call for your function.
* Build Composer functions by using https://docs.mulesoft.com/dataweave/2.3/dw-operators[DataWeave].

Reply:
That actually changes the meaning a bit. I can combine it with the above paragraph and say: When building formulas for a flow, keep in mind that:

* You can nest functions and use the input parameters of another function call for your function.
* Composer functions used in Composer are built using https://docs.mulesoft.com/dataweave/2.3/dw-operators[DataWeave].